Competition2017/src/competition/subsystems/servo/SetServoCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.servo;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.XPropertyManager;
+
+public class SetServoCommand extends BaseCommand {
+    private ServoSubsystem servoSubsystem;
+    private double targetPose;
+    
+    @Inject
+    public SetServoCommand(XPropertyManager propMan, ServoSubsystem servoSubsystem){
+        this.servoSubsystem = servoSubsystem;
+        this.requires(servoSubsystem);
+    }
+
+    @Override
+    public void initialize() {}
+
+    @Override
+    public void execute() {
+        servoSubsystem.setServo(targetPose);
+    }
+    
+    public void setTargetPose(double value) {
+        targetPose = value;
+    }
+
```

Comment:
I think it would make sense to have it end immediately so it can be quickly chained into a `CommandGroup`. In my use case (which I believe is the only one), I want to tell the servo to move into the "engaged" position and continue doing other things in the autonomous program without delay.